### PR TITLE
display pull request result #6

### PR DESCRIPTION
**Pull request by windwp**

### CODE REVIEW

**Comment by windwp** [comment]
@tami5

**Comment by windwp** [comment]
@tami5

**Comment by tami5** [comment]
It worked, but instead of opening qf, it just staged the local changes I have and switch branch then printed completed :D in other cases I got `E5108: Error executing lua .../site/pack/packer/start/plenary.nvim/lua/plenary/job.lua:461: 'gh pr checkout 85' was unable to complete in 5000 ms` but it still switched

don't know what but if there not committed changes or not unstaged changes should it still switch/check out that pr?.

Any way its better now. Thanks @windwp

**Comment by windwp** [comment]
I don't know what happen but this command "gh pr checkout" don't stage a local changes
I increase timeout to 10000ms  Maybe it is not enough if your pull request has too many file but i think it will be fine with 10000ms

File: lua/telescope/_extensions/gh_actions.lua [hunk 1]
```diff
@@ -13,14 +13,19 @@ A.gh_pr_checkout = function(prompt_bufnr)
     return
   end
 
-  local qf_entry={}
+  local qf_entry = {{
+      text="Checkout pull request, Please wait."
```
Comment (tami5) [inline lua/telescope/_extensions/gh_actions.lua:17]: Checking out pull request #, please wait ...

**Comment by Conni2461** [comment]
If you are already working on this project right now can you please stop injecting functions into builtin like i suggested here: https://github.com/nvim-telescope/telescope-github.nvim/pull/3#discussion_r551010625
We now have auto completion for extensions as well

**Pull request merged**